modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/CheckboxMultiple/CheckboxMultiple.es.js

```diff
@@ -66,13 +68,17 @@ const CheckboxMultiple = ({
 	options,
 	predefinedValue,
 	required,
+	valid,
 	value: initialValue,
 }) => {
 	const [value, setValue] = useState(initialValue);
 
 	const displayValues = value && value.length > 0 ? value : predefinedValue;
 	const Toggle = isSwitcher ? Switcher : ClayCheckbox;
 
+	const invalid = (displayErrors && errorMessage && !valid ) ? 'true' : 'false';
+	const describedBy = (displayErrors && errorMessage && !valid ) ? 'errorMessage' : null;
```

Comment:
@rijema, why didn't you re-use the checking `(displayErrors && errorMessage && !valid )` to set the values of `invalid` and `describedBy`?

Something like:
```
if (displayErrors && errorMessage && !valid) {
    const invalid = 'true';
    const describedBy = 'errorMessage';
}
```

Reply:
In this part, my goal was to update the value every time the page is rendered. With a new reload of the page, and not with an instant change, the value would be recalculated properly, that's why I used ternary operator, but yeah, it doesn't explain why I verify two times the same thing.

I couldn't declare and add a value in a conditional in this case because I use these values even if it not pass in the verification, avoiding a Javascript error reference. 

I think that my explanation was very confusing, so I'm sorry for that.

I added a new commit making changes searching achieve your review on this, but instead of const I used a let declaration. 

```
        let invalid = 'false';
	let describedBy = null;

	if (displayErrors && errorMessage && !valid) {
   	 	invalid = 'true';
    	       describedBy = 'errorMessage';
	}
```

Let me know if you want me to update this pull request with a new commit or if I have to squash in some of these.